SimPEG/electromagnetics/frequency_domain/sources.py

```diff
@@ -533,12 +533,14 @@ class CircularLoop(MagDipole):
 
     current = properties.Float("current in the loop", default=1.0)
 
+    N = properties.Integer("number of turns in the loop", default=1)
```

Comment:
```suggestion
    n_turns = properties.Integer("number of turns in the loop", default=1)
```

would it work with you if we update the naming here? `n_turns` is a bit more descriptive (and pep8)